packages/graphback-codegen-resolvers/src/plugins/ResolverGeneratorPlugin.ts

```diff
@@ -0,0 +1,86 @@
+import { getModelTypesFromSchema, GraphbackCoreMetadata, GraphbackPlugin, TypeScriptResolversVisitor } from '@graphback/core';
+import { buildASTSchema, GraphQLField, GraphQLFieldMap, GraphQLObjectType, GraphQLOutputType, GraphQLSchema, isListType, isNonNullType, parse, printSchema, visit } from 'graphql';
+import { ResolverGeneratorOptions } from '../api/ResolverGeneratorOptions';
+import * as pluralize from "pluralize";
+
+export interface ResolverDefinition {
```

Comment:
We do not need those types as generally resolvers are not type safe. 
TypeSafety will be coming externally from codegen

Reply:
This is purely for building a collection of resolvers which can then be piped to a file at the very end.